arangod/Agency/AsyncAgencyComm.cpp

```diff
@@ -486,7 +489,7 @@ AsyncAgencyComm::FutureResult AsyncAgencyComm::sendWithFailover(
               << "] had exception during request";
           try {
             e.throwIfFailed();
-          } catch (const std::exception& e) {
+          } catch (std::exception const& e) {
```

Comment:
Just for info: I really want to update clang format to fix const west/east issue automatically